src/@types/global.d.ts

```diff
@@ -0,0 +1,2 @@
+declare type Maybe<T> = T | null | undefined;
+declare type HTTPMethod = 'GET' | 'POST' | 'PUT' | 'DELETE'
```

Comment:
This types are "global". They can be referenced without the need to manually import them beforehand